src/examples/zap-baseline.py

```diff
@@ -349,6 +355,9 @@ def main(argv):
       logging.debug ('Ajax Spider complete')
 
     # Wait for passive scanning to complete
+    while (datetime.now() - start).seconds < ((mins * 60) + 10):
```

Comment:
OK, rereading the PR description I see why you are using the spider limit (but still don't know why it's needed a new check/wait).